agent/lib/kontena/workers/container_log_worker.rb

```diff
@@ -67,7 +68,7 @@ def on_message(id, stream, chunk)
         type: stream,
         data: data
       }
-      @queue << msg
+      @queue << msg if @queue.size < QUEUE_LIMIT
```

Comment:
Not sure if that is a good idea.. then it will also skip fluentd shipping. 

I did some refactoring and now LogWorker is responsible of throttling.. it will terminate actual container log workers if queue grows too big, and start them again when queue is processed.